lib/Extractor/KLEEBuilder.cpp

```diff
@@ -141,6 +142,8 @@ class KLEEBuilder : public ExprBuilder {
       }
       return E;
     }
+    case Inst::Freeze:
+      return get(Ops[0]);
```

Comment:
is this just ignoring freeze in kleebuilder?

Reply:
Sort of. I cannot put a explicit assert here to crash (as you suggested) as it results in several test failures because freeze is also synthesized now. So, it does need some sort of klee support for those RHSes to interpret successfully.

Reply:
we don't want silent unsound behavior, so something besides this needs to be done here

Reply:
you could add a flag to not synthesize freeze, and use it in the tests that crash